### ISSUE TITLE
ansible-lint schema check fails for doc_changes with network collections

### ISSUE THREAD

**Issue by KB-perByte** [ansible/ansible-lint#3261]
lint fails for _doc_changes_
valid section across Ansible network modules 
https://github.com/ansible-collections/cisco.ios/blob/main/changelogs/config.yaml#L29
##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

```console (paste below)
ansible-lint --version
```

- ansible installation method: one of source, pip, OS package
- ansible-lint installation method: one of source, pip, OS package

##### STEPS TO REPRODUCE

```console (paste below)

```

##### Desired Behavior

Possible security bugs should be reported via email to `security@ansible.com`

##### Actual Behavior

Please give some details of what is happening.
Include a [minimum complete verifiable example] with:

- minimized playbook to reproduce the error
- the output of running ansible-lint including the command line used
- if you're getting a stack trace, also the output of
  `ansible-playbook --syntax-check playbook`

```paste below
WARNING  Listing 1 violation(s) that are fatal
schema[changelog]: Additional properties are not allowed ('doc_changes' was unexpected)
changelogs/changelog.yaml:1  Returned errors will not include exact line numbers, but they will mention
the schema name being used as a tag, like ``schema[playbook]``,
``schema[tasks]``.

This rule is not skippable and stops further processing of the file.

If incorrect schema was picked, you might want to either:

* move the file to standard location, so its file is detected correctly.
* use ``kinds:`` option in linter config to help it pick correct file type.


Error: Additional properties are not allowed ('doc_changes' was unexpected)
Read documentation for instructions on how to ignore specific rule violations.

                Rule Violation Summary                
 count tag               profile rule associated tags 
     1 schema[changelog] basic   core  
```

[minimum complete verifiable example]: http://stackoverflow.com/help/mcve
